plone/app/event/bbb/portlets/portlet_calendar.py

```diff
@@ -1,19 +1,27 @@
 from Products.Five.browser.pagetemplatefile import ViewPageTemplateFile
 from plone.app.portlets import PloneMessageFactory as _
-from plone.app.portlets.browser.z3cformhelper import AddForm
-from plone.app.portlets.browser.z3cformhelper import EditForm
+try:
+    from plone.app.portlets.browser import z3cformhelper
+    P_A_PORTLETS_PRE_3 = True
+except:
```

Comment:
explicit ``ImportError`` catching would be better, but even this may hide circular dependency problems. Why dont you just check the version against the value of ``pkg_resources.require("plone.app.portlets")[0].version`` ?